recipes-kernel/jool/jool_git.bb

```diff
@@ -21,16 +21,12 @@ EXTRA_OEMAKE += 'ARCH="${ARCH}" CROSS_COMPILE="${TARGET_PREFIX}" \
 do_compile() {
     # Only build the kernel modules
     unset CFLAGS CPPFLAGS CXXFLAGS LDFLAGS
-    oe_runmake -C "${S}/mod/stateful" CC="${KERNEL_CC}" LD="${KERNEL_LD}" \
-            AR="${KERNEL_AR}" O=${STAGING_KERNEL_BUILDDIR}
-    oe_runmake -C "${S}/mod/stateless" CC="${KERNEL_CC}" LD="${KERNEL_LD}" \
+    oe_runmake -C "${S}/src/mod/nat64" CC="${KERNEL_CC}" LD="${KERNEL_LD}" \
```

Comment:
Do we need to drop jool siit as well here?

Reply:
I'll re-add jool_siit.